doc/API.md

```diff
@@ -67,16 +67,35 @@ Get an address to build a deposit transaction.
 
 ### Vault resource
 
-| Field         | Type   | Description                                                                          |
-| ------------- | ------ | ------------------------------------------------------------------------------------ |
-| `amount`      | int    | Amount of the vault in satoshis                                                      |
-| `status`      | string | Status of the vault (see [vault statuses](#vault-statuses))                          |
-| `txid`        | string | Deposit txid of the vault deposit transaction                                        |
-| `vout`        | vout   | Index of the deposit output in the deposit transaction.                              |
+| Field         | Type   | Description                                                 |
+| ------------- | ------ | ----------------------------------------------------------- |
+| `amount`      | int    | Amount of the vault in satoshis                             |
+| `blockheight` | int    | Blockheight of the deposit transaction block                |
+| `created_at`  | int    | Timestamp of the deposit transaction block                  |
+| `status`      | string | Status of the vault (see [vault statuses](#vault-statuses)) |
+| `txid`        | string | Deposit txid of the vault deposit transaction               |
+| `updated_at`  | int    | Timestamp of the last status change                         |
+| `vout`        | int    | Index of the deposit output in the deposit transaction.     |
+
+Some fields can be added:
+
+| Field         | Type                                                      | Description                                                                |
+| ------------- | --------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `final_tx`    | [Vault transaction resource](#vault-transaction-resource) | Final transaction -- not present if the vault is not spent or cancelled    |
+| `unvault_tx`  | [Vault transaction resource](#vault-transaction-resource) | Unvault transaction -- not present if vault is not unvaulting or unvaulted |
```

Comment:
I wonder if it should be split instead: optional fields are IMHO non-natural. Looks like you are describing executed transactions here so maybe a `listconchaintransactions [vault_id]` call ?

Wouldn't return anything for the unconfirmed vaults, the deposit for the funded, the unvault for the unvaulted, the spend too if spent, or the cancel.